test/transcoding/opaque-pointer-annotation.ll

```diff
@@ -0,0 +1,35 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
I'll remove this test just updating the one from https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1628 when the latest is merged.